src/systemd.rs

```diff
@@ -33,21 +33,7 @@ pub fn stop_systemd_target() {
 ///Determine if systemd is used as the init system. This should work on all linux distributions.
 pub fn is_systemd_used() -> &'static bool {
 	static IS_SYSTEMD_USED: OnceLock<bool> = OnceLock::new();
-	IS_SYSTEMD_USED.get_or_init(|| {
-		match Command::new("ls").args(["/run/systemd/system"]).output() {
-			Ok(output) => {
-				if output.status.success() {
-					true
-				} else {
-					false
-				}
-			}
-			Err(error) => {
-				warn!("unable to check if systemd is used: {}", error);
-				false
-			}
-		}
-	})
+	IS_SYSTEMD_USED.get_or_init(|| std::fs::metadata("/run/systemd/system").is_ok())
```

Comment:
Why metadata instead of `std::path::Path::exists`?

Reply:
The functionality is equivalent: https://doc.rust-lang.org/stable/src/std/path.rs.html#2872-2874

Reply:
That is an implementation detail of `std` though. It may or may not keep being the same, we should use the simpler method.

Reply:
Done.